static/app/components/events/interfaces/crashContent/exception/utils.spec.tsx

```diff
@@ -0,0 +1,121 @@
+import {render} from 'sentry-test/reactTestingLibrary';
+
+import {Linkify} from 'sentry/components/events/interfaces/crashContent/exception/utils';
+
+describe('Linkify()', function () {
+  it('does not alter text that does not contain urls', function () {
+    const text = 'This is not a link';
+    const {container, getByText} = render(<Linkify exceptionText={text} />);
+
+    expect(getByText(text)).toBeInTheDocument();
+
+    // check if the text is directly within the container (which implies a fragment)
+    expect(container.firstChild).toHaveTextContent(text);
+  });
+
+  it('returns an empty string when no text is provided', function () {
+    const text = '';
+    const {container} = render(<Linkify exceptionText={text} />);
+
+    expect(container.firstChild).toHaveTextContent(text);
+  });
+
+  it('applies links to text containing a single url', function () {
+    const url = 'https://www.example.com';
+    const text = `Go to ${url} to search.`;
+    const {container, getByText} = render(<Linkify exceptionText={text} />);
+
+    const linkElement = getByText(url);
```

Comment:
Small thing, but it's preferred to use `screen.getByRole` when possible. Here it would be `screen.getByRole('link', { name: url })`